.github/workflows/security_analysis.yml

```diff
@@ -35,5 +35,5 @@ jobs:
       - name: Vulnerability Scan Go Code
         uses: Templum/govulncheck-action@v1.0.0
         with:
-           go-version: 1.20.6
+           go-version: 1.21.4
```

Comment:
This is pinned to a specific version, all the other actions are using the latest patch version of 1.21. Since we have the tests to prove everything works i think it would be better to use `^1.21`

Reply:
In kubesec we just changed the go action to read the go module then you dont have to bump this anymore

Reply:
I did wonder. Let me look into doing that and I'll resubmit. Thanks both :heart: